cache/converter.go

```diff
@@ -90,6 +105,10 @@ func getConverter(ctx context.Context, cs content.Store, desc ocispecs.Descripto
 			}
 			return &readCloser{r, ra.Close}, nil
 		}
+	case compression.NydusBootstrap:
+		return func(ctx context.Context, cs content.Store, desc ocispecs.Descriptor) (*ocispecs.Descriptor, error) {
+			return doCompression(ctx, ref, comp, s)
+		}, nil
```

Comment:
If the `*cacheRecord.getBlob()` is nydus blob, the cache always computes the blob from scratch every time a compresion variant is required (e.g. the client requires nydus first, then gzip and then zstd)? I think it should be deduplicated.

Reply:
Thanks @ktock for the reply.

It is not true. The `*cacheRecord.getBlob()` always gets `nydus bootstrap` instead of `nydus blob` if the client requires nydus first.